lunches.py

```diff
@@ -373,6 +373,23 @@ def rusty_bell_pub(dom):
         yield food
 
 
+@restaurant("Viktorka", "https://www.viktorkaostrava.cz/denni-menu/", Location.Poruba)
+def viktorka(dom):
+    sections = dom.css("ul.elementor-price-list")
+    type = [Soup, Lunch]
+    day_nth = datetime.datetime.today().weekday()
+    for i in range(2):
```

Comment:
`for i, food_type in enumerate(types)` so we dont need to access `types` by index and additionally have magic constant `2` here

Reply:
done